packages/ricos-content/web/src/RicosContentAPI/metadata-utils.ts

```diff
@@ -0,0 +1,36 @@
+import { cloneDeep, merge } from 'lodash';
+import { createEmptyContent } from './createEmptyContent';
+import { RichContent } from 'ricos-schema';
+import { v4 as uuid } from 'uuid';
+
+/**
+ * Duplicates `RichContent` object, and assigning a newly generated `id`
+ * @param content = `RichContent` object
+ * @returns Cloned object with new `id`
+ */
+export const duplicateContent = (content: RichContent): RichContent =>
+  merge(cloneDeep(content), { metadata: { id: uuid() } });
```

Comment:
Separate functionality (adder of ID + duplicator).
Also remove `merge`
```suggestion
 { ...cloneDeep(content), metadata: { ...content.metadata, id: uuid() } };
```